src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -53,7 +55,11 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
   const [showCopied, setShowCopied] = useState(false);
   const [compatibilityData, setCompatibilityData] = useState({});
   const [canUseShareableUrls, setCanUseShareableUrls] = useState(true);
```

Comment:
nit

```suggestion
const [isShareableUrlsEnabled, setIsShareableUrlsEnabled] = useState(true);
```

Reply:
hmm i don't think it's about enabled or not, how about `[isCompatible, setIsCompatible]` ? wdyt @rashidakanchwala